### PR TITLE
February redirects

### PR DESCRIPTION
**Pull request by mgrassotti**
https://trello.com/c/v40TaWia/620-february-re-directs-small

Reslug “her-majestys-magistrates-courts-servuce-inspectorate” as
requested in February redirects

### CODE REVIEW

**Comment by mgrassotti** [comment]
This is related to https://github.gds/gds/router-data/pull/635

File: db/data_migration/20170303121757_her_majestys_magistrates_courts_servuce_inspectorate_slug_change.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'data_hygiene/organisation_reslugger'
+
+organisation = WorldwideOrganisation.find_by(slug: "her-majestys-magistrates-courts-servuce-inspectorate")
```
Comment (gpeng) [inline db/data_migration/20170303121757_her_majestys_magistrates_courts_servuce_inspectorate_slug_change.rb]: This isn't a `WorldwideOrganisation` it's an `Organisation`

Comment (mgrassotti) [inline db/data_migration/20170303121757_her_majestys_magistrates_courts_servuce_inspectorate_slug_change.rb]: 👍

**Comment by mgrassotti** [comment]
👍

**Pull request merged**